repose-aggregator/core/repose-core/src/main/java/org/openrepose/core/services/deploy/ContainerConfigurationListener.java

```diff
@@ -56,14 +56,11 @@ public synchronized void configurationUpdated(ContainerConfiguration configurati
                 if (ad.getCheckInterval() > 0) {
                     dirWatcher.updateCheckInterval(ad.getCheckInterval());
                 }
-
-                dirWatcher.updateArtifactDirectoryLocation(new File(ad.getValue()));
-
-                deploymentDirectory = new File(dd.getValue());
+                dirWatcher.updateArtifactDirectoryLocation(new File(ad.getValue().trim()));
+                deploymentDirectory = new File(dd.getValue().trim());
```

Comment:
Having a fully qualified path that starts our ends with a space is bad practice on any OS. The `trim()` doesn't effect whitespace in the middle, so it shouldn't limit Repose in any way.